jinahub/indexers/query/vector/AnnoyIndexer/tests/test_annoyindexer.py

```diff
@@ -51,139 +36,12 @@ def test_simple_annoy():
-        indexer.update(['4'], np.array([[200, 200, 200]]))
-        indexer.save()
-        assert indexer.size == 4
-
-    with BaseIndexer.load(save_abspath) as indexer:
-        assert isinstance(indexer, AnnoyIndexer)
-        idx, dist = indexer.query(queries, top_k=3)
-        np.testing.assert_equal(idx, np.array([[5, 6, 4], [5, 6, 4], [6, 5, 4], [7, 4, 6]]).astype(str))
-        assert idx.shape == dist.shape
-        assert idx.shape == (4, 3)
-
-    # delete
-    with BaseIndexer.load(save_abspath) as indexer:
-        indexer.delete(['4'])
-        indexer.save()
-        assert indexer.size == 3
-
-    with BaseIndexer.load(save_abspath) as indexer:
-        assert isinstance(indexer, AnnoyIndexer)
-        idx, dist = indexer.query(queries, top_k=2)
-        np.testing.assert_equal(idx, np.array([[5, 6], [5, 6], [6, 5], [7, 6]]).astype(str))
-        assert idx.shape == dist.shape
-        assert idx.shape == (4, 2)
-
-
-def test_annoy_indexer_known_big(metas):
-    """Let's try to have some real test. We will have an index with 10k vectors of random values between 5 and 10.
-     We will change tweak some specific vectors that we expect to be retrieved at query time. We will tweak vector
-     at index [0, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9000], this will also be the query vectors.
-     Then the keys will be assigned shifted to test the proper usage of `int2ext_id` and `ext2int_id`
-    """
-    vectors = np.random.uniform(low=5.0, high=10.0, size=(10000, 1024))
-
-    queries = np.empty((10, 1024))
-    for idx in range(0, 10000, 1000):
-        array = idx * np.ones((1, 1024))
-        queries[int(idx / 1000)] = array
-        vectors[idx] = array
-
-    keys = np.arange(10000, 20000).reshape(-1, 1).astype(str)
-    with AnnoyIndexer(index_filename='annoy.test.gz', n_trees=5000, metas=metas) as indexer:
-        indexer.add(keys, vectors)
-        indexer.save()
-        assert os.path.exists(indexer.index_abspath)
-        save_abspath = indexer.save_abspath
-
-    with BaseIndexer.load(save_abspath) as indexer:
-        assert isinstance(indexer, AnnoyIndexer)
-        idx, dist = indexer.query(queries, top_k=1)
-        print(f' idx {idx}')
-        np.testing.assert_equal(idx, np.array(
-            [[10000], [11000], [12000], [13000], [14000], [15000], [16000], [17000], [18000], [19000]]).astype(str))
-        assert idx.shape == dist.shape
-        assert idx.shape == (10, 1)
-        np.testing.assert_equal(indexer.query_by_key(['10000', '15000']), vectors[[0, 5000]])
+def test_query_vector(tmpdir):
+    metas = {'workspace': str(tmpdir), 'name': 'dbms', 'pea_id': 0, 'replica_id': 0}
+    indexer = AnnoyIndexer(dump_path='tests/dump1', num_dim=7, top_k=5, metas=metas)
+    docs = DocumentArray([Document(embedding=np.random.random(7))])
+    TOP_K = 5
+    indexer.search(docs)
+    assert len(docs) == 1
+    assert len(docs[0].matches) == TOP_K
+    assert len(docs[0].matches[0].embedding) == 7
```

Comment:
Can you add more asserts here?

- id is one from the dump
- embedding is one from the dump
- score is a specific number (use np.random.seed to get reproducible embeddings for your `docs`)

Reply:
sure. But actually, this is also almost the same with numpy indexer 
https://github.com/jina-ai/indexers/blob/f410010ef1ae64d825d7a37fb8158f03e8eb9b0b/jinahub/indexers/query/vector/NumpyIndexer/tests/test_query.py#L10